suites/quincy/rbd/tier-2_rbd_upgrade_5x_to_6x.yaml

```diff
@@ -0,0 +1,160 @@
+#===============================================================================================
+# Tier-level: 2
+# Test-Suite: tier-2_rbd_mirror_snapshot_regression.yaml
```

Comment:
this suite name is not matching with test suite.

Reply:
Updated, please check
Sorry for the silly miss and thanks for pointing out